<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>username_sailer</class>
 <widget class="QWidget" name="username_sailer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="oldPS">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>146</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 147, 147);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="oldPSLabel">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>156</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>旧密码：</string>
   </property>
  </widget>
  <widget class="QLabel" name="differPS">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>346</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码不一致</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPS">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>226</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 147, 147);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="newPSLable2">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>291</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重新确认：</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrongPS">
   <property name="geometry">
    <rect>
     <x>501</x>
     <y>156</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码错误</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>331</x>
     <y>386</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Alibaba PuHuiTi</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="newPSLable">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPS2">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>286</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(147, 147, 147);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="changeFail">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改失败</string>
   </property>
  </widget>
  <widget class="QLabel" name="changeSuccess">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改成功</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
